modules/configuration/ajax/process.php

```diff
@@ -27,39 +27,103 @@
 $client = new NDB_Client();
 $client->makeCommandLine();
 $client->initialize();
-
-$DB =& Database::singleton();
+$factory = NDB_Factory::singleton();
```

Comment:
sorry, hopefully one last thing! you want a \ before NDB_Factory. for the same lines below aswell